charts/finrem-ccd-definitions/values.preview.template.yaml

```diff
@@ -25,15 +25,15 @@ ccd:
       value: ${API_GATEWAY_IDAM_SECRET}
 
   userProfileApi:
-    authorisedServices: ccd_admin,ccd_data,ccd_definition,divorce
+    authorisedServices: ccd_admin,ccd_data,ccd_definition
```

Comment:
Let's keep this in mind in case the communication between finrem and this instance of CCD fails